whirlpool/eventsocket.py

```diff
@@ -14,20 +14,42 @@
 
 RECV_MSG_MATCHER = re.compile("{(.*)}\\x00")
 
+# closing frame status codes for websocket.
+STATUS_NORMAL = 1000
+STATUS_GOING_AWAY = 1001
+STATUS_PROTOCOL_ERROR = 1002
+STATUS_UNSUPPORTED_DATA_TYPE = 1003
+STATUS_STATUS_NOT_AVAILABLE = 1005
+STATUS_ABNORMAL_CLOSED = 1006
+STATUS_INVALID_PAYLOAD = 1007
+STATUS_POLICY_VIOLATION = 1008
+STATUS_MESSAGE_TOO_BIG = 1009
+STATUS_INVALID_EXTENSION = 1010
+STATUS_UNEXPECTED_CONDITION = 1011
+STATUS_SERVICE_RESTART = 1012
+STATUS_TRY_AGAIN_LATER = 1013
+STATUS_BAD_GATEWAY = 1014
+STATUS_TLS_HANDSHAKE_ERROR = 1015
+STATUS_UNAUTHORIZED = 3000
+
+RECONNECT_TIMEOUT = 30
```

Comment:
```suggestion
RECONNECT_DELAY = 30
```
It seems like a delay to reconnect, and not a connection timeout, right?